init/main.go

```diff
@@ -619,6 +622,15 @@ func boost() error {
 		return err
 	}
 
+	// Mount efivarfs if running in EFI mode
+	if _, err := os.Stat("/sys/firmware/efi"); errors.Is(err, os.ErrNotExist) {
```

Comment:
BTW what is the difference of this construction from `os.IsNotExist(err)`?

Reply:
It is the same, wrote it this way as [go documentation](https://pkg.go.dev/os@go1.17#IsNotExist) recommended it. If it is preferred I can change it to `os.isNotExist(err)`

Reply:
Ack. Let's follow docs recommendation. It says `errors.Is(err, fs.ErrNotExist)` should be used.

TODO for myself: change other places in booster and use `errors.Is` to be consistent with the documentation.